rosetta-lang/src/main/java/com/regnosys/rosetta/validation/ImportManagementService.java

```diff
@@ -0,0 +1,109 @@
+package com.regnosys.rosetta.validation;
+
+import com.regnosys.rosetta.RosettaEcoreUtil;
+import com.regnosys.rosetta.rosetta.Import;
+import com.regnosys.rosetta.rosetta.RosettaModel;
+import com.regnosys.rosetta.rosetta.RosettaRootElement;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.eclipse.xtext.naming.IQualifiedNameProvider;
+import org.eclipse.xtext.naming.QualifiedName;
+
+public class ImportManagementService {
```

Comment:
This is really good!

Reply:
In terms of package, how about putting it in the `utils` package? It's not necessarily related to validation.